apps/marketing/content/blog/announcing-open-source-docusign-alternative.mdx

```diff
@@ -17,32 +17,32 @@ tags:
 
 ## How do we build trusted systems?
 
 While doing research for Documenso I came upon a quote that expresses the current state of document signing pretty well:
 
 > Document signing is NOT a technical problem. [Editor’s Note: Because it was solved technically a long time ago] It’s a legal acceptance problem — and everyone KNOWS DocuSign and friends and understands how they’re admissible. Anything else would have to compete with that and people would be suspicious of it for a long time.
 
-While this may sound like a hurdle at first, it immediately gave me a sense of validation for a more open approach to signing. People will and should be suspicious of their tools and demand a high bar when it comes to trust. And the way to earn this trust is by being open. Trusted tools should be the result of thoughtful discussion and reviews. They should be the result of the needs and will of its community. They should be transparent, adaptable, and empowering while using. Open-Source embodies these values very well for software, which makes it a perfect fit for this space and creating a high-trust tool.
+While this may sound like a hurdle at first, it immediately gave me a sense of validation for a more open approach to signing. People will and should be suspicious of their tools and demand a high bar when it comes to trust. And the way to earn this trust is by being open. Trusted tools should be the result of thoughtful discussion and reviews. They should be the result of the needs and will of its community. They should be transparent, adaptable, and empowering while using. Open source embodies these values very well for software, which makes it a perfect fit for this space and creating a high-trust tool.
 
 ## Next Steps
 
-So, what can you expect from here on out? I have started to build Documenso 0.1 which is scheduled to release in “early” 2023. If you are interested in helping make this happen, let me know via hi@documenso.com. Getting working code into the hands of the perspective Documenso community is currently the #1 goal. Other than that I will be releasing several articles about document signing and what something like Documenso should look like, in my humble opinion. So stay tuned!
+So, what can you expect from here on out? I've started to build Documenso 0.1 which is scheduled to release in “early” 2023. If you're interested in helping make this happen, let me know via `hi@documenso.com`. Getting working code into the hands of the perspective Documenso community is currently the #1 goal. Other than that I'll be releasing several articles about document signing and what something like Documenso should look like, in my humble opinion. So stay tuned!

```

Comment:
Should be `[hi@documenso.com](mailto:hi@documenso.com)` no?

Reply:
or `timur@`? do you mind if we add your email address @ElTimuro?

Reply:
I'll say leave it at `hi@documenso.com` for now it's a shared inbox so it gives us both a chance to chat with any interested parties.

Reply:
fixed!